server/sonar-server/src/main/java/org/sonar/server/exceptions/BadRequestException.java

```diff
@@ -32,11 +32,19 @@
 
   private final transient Errors errors;
 
+  /**
+   * @deprecated Bundle key should be replaced by real message as only english is returned on server side errors
+   */
+  @Deprecated
   public BadRequestException(String l10nKey, Object... l10nParams) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)